lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
For 2., yes there are new feature changes as well, but I don't think they are the decisive factors for major version upgrade.

For 1., what's the condition that the ```-preview``` suffix on the client version string can be removed, and what future conditions do we need to keep? If it's not clear, we tend not to remove it yet.